apps/sasquatch/src/projectDependency/java/com/microsoft/appcenter/sasquatch/activities/AppDocumentDetailActivity.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.appcenter.sasquatch.activities;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.TextView;
+
+import com.microsoft.appcenter.sasquatch.R;
+
+public class AppDocumentDetailActivity extends AppCompatActivity {
+
+    private TextView mDocumentId;
+
+    private TextView mDocumentContents;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_app_document_detail);
+
+        mDocumentId = findViewById(R.id.app_document_id);
+        mDocumentContents = findViewById(R.id.app_document_content);
```

Comment:
Actually I found issue for the first task: app documents.

The contents is empty, also this file has warnings.

Reply:
@guperrot hi yes, I know the app document content is empty, because I don't know what is to display on the content. The document in log is like this 

"Documents":[{"document":{"test":"ABC"},"id":"readonly-doc-001","PartitionKey":"readonly","_rid":"tVQZAP4NhY8FAAAAAAAAAA==","_self":"dbs\/tVQZAA==\/colls\/tVQZAP4NhY8=\/docs\/tVQZAP4NhY8FAAAAAAAAAA==\/","_etag":"\"0000667a-0000-0000-0000-5c9af1180000\"","_attachments":"attachments\/","_ts":1553658136},{"document":{"test":"ABC"},"id":"readonly-doc-002","PartitionKey":"readonly","_rid":"tVQZAP4NhY8GAAAAAAAAAA==","_self":"dbs\/tVQZAA==\/colls\/tVQZAP4NhY8=\/docs\/tVQZAP4NhY8GAAAAAAAAAA==\/","_etag":"\"0000747a-0000-0000-0000-5c9af1280000\"","_attachments":"attachments\/","_ts":1553658152},{"id":"doc1","document":{"key":"Hello Zakeel!"},"PartitionKey":"readonly","_rid":"tVQZAP4NhY9lAAAAAAAAAA==","_self":"dbs\/tVQZAA==\/colls\/tVQZAP4NhY8=\/docs\/tVQZAP4NhY9lAAAAAAAAAA==\/","_etag":"\"0200caa0-0000-0000-0000-5c9ea1a90000\"","_attachments":"attachments\/","_ts":1553899945}],"

and I can get the document like this documents.getCurrentPage().getItems().get(i).getDocument(), but perhaps it is not the content we want.

Reply:
@guperrot Hi Guperrot, I have fixed it by function Utils.getGson().toJson, and app document content UI like below:

![0408-1](https://user-images.githubusercontent.com/46912152/55717676-6de07c00-5a2c-11e9-89b9-82b97ddaf941.gif)

Reply:
> @guperrot Hi Guperrot, I have fixed it by function Utils.getGson().toJson, and app document content UI like below:
> 
> ![0408-1](https://user-images.githubusercontent.com/46912152/55717676-6de07c00-5a2c-11e9-89b9-82b97ddaf941.gif)

Yes that is what was specified in the task mock-ups to just use JSON for now.